library/std/src/error.rs

```diff
@@ -831,6 +909,18 @@ impl dyn Error + 'static {
             None
         }
     }
+
+    /// Request a reference to context of type `T`.
+    #[unstable(feature = "error_generic_member_access", issue = "none")]
+    pub fn request_ref<T: ?Sized + 'static>(&self) -> Option<&T> {
+        core::any::request_ref(self)
+    }
+
+    /// Request a value to context of type `T`.
```

Comment:
"value to context" sounds weird. (repeated multiple times in this PR)

Reply:
I've rewritten these as "Request a value of type `T` as context about this error", how does that sound compared to the original version?

Reply:
Seems reasonable!